### PR TITLE
Release 0.18.0

### PR DESCRIPTION
**Pull request by r3nic1e**

### CODE REVIEW

File: .github/workflows/go-tests.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
           registry: false
 
       - name: Set up Helm
-        uses: azure/setup-helm@v1
+        uses: azure/setup-helm@v2.0
         with:
           version: v3.7.1
```
Comment (zhilyaev) [inline .github/workflows/go-tests.yml:27]: setup helm 3.8.0

File: .golangci.yml [hunk 2]
```diff
@@ -293,20 +292,69 @@ linters-settings:
...
     # Select the Go version to target. The default is '1.13'.
     go: "1.17"
     # https://staticcheck.io/docs/options#checks
-    checks: [ "all", "-ST1000", "-ST1003", "-ST1016", "-ST1017", "-ST1020", "-ST1021", "-ST1022" ]
+    checks:
+      [
+        "all",
+        "-ST1000",
+        "-ST1003",
+        "-ST1016",
```
Comment (zhilyaev) [inline .golangci.yml:306]: I think that will normal yaml list

```yaml
checks:
       - "-ST1003",
       - "-ST1016",
```

**Review by zhilyaev — DISMISSED** [review]

File: .golangci.yml (hunk 2 above)
Comment (r3nic1e) [reply .golangci.yml]: fixed

File: .github/workflows/go-tests.yml (hunk 1 above)
Comment (r3nic1e) [reply .github/workflows/go-tests.yml]: removed it completely - it was used to create repositories file

**Review by zhilyaev — APPROVED** [review]

**Pull request merged**